package.json

```diff
@@ -18,7 +18,7 @@
     "ast-types": "git+https://github.com/jlongster/ast-types.git",
     "babylon": "^6.15.0",
     "esutils": "^2.0.2",
-    "flow-parser": "^0.37.0",
+    "flow-parser": "0.38.0",
```

Comment:
Do you mind if we merge this as is and then I send a pull request to remove all the `^` at once?